modules/executors/startup-script.sh.tpl

```diff
@@ -38,6 +43,7 @@ EXECUTOR_NUM_TOTAL_JOBS="$${EXECUTOR_NUM_TOTAL_JOBS}"
 EXECUTOR_MAX_ACTIVE_TIME="$${EXECUTOR_MAX_ACTIVE_TIME}"
 EXECUTOR_USE_FIRECRACKER="$${EXECUTOR_USE_FIRECRACKER}"
 $${STARTUP_SCRIPT_LINE}
+$${DOCKER_REGISTRY_NODE_EXPORTER_URL_LINE}
```

Comment:
Would it not be fine to just have it as `DOCKER_REGISTRY_NODE_EXPORTER_URL=$${DOCKER_REGISTRY_NODE_EXPORTER_URL}`, given itll be ignored if its `''` anyway

Reply:
Yeah it wouldn't matter but.. also it doesn't hurt to have a clean config, does it?